pkgs/development/python-modules/torchaudio/bin.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, isPy37
+, isPy38
+, isPy39
+, patchelf
+, python
+, pytorch-bin
+}:
+
+let
+  pyVerNoDot = builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+  srcs = import ./binary-hashes.nix version;
+  unsupported = throw "Unsupported system";
+  version = "0.10.0";
+in buildPythonPackage {
```

Comment:
It would be nice if we could inline some of these things.

Reply:
What do you mean? Does it mean to make `binary-hashes.nix` inline?

Reply:
I mean line 14 to 17.